packages/flutter_tools/lib/src/base/common.dart

```diff
@@ -21,5 +21,12 @@ class ToolExit implements Exception {
   final int? exitCode;
 
   @override
-  String toString() => 'Exception: $message'; // TODO(ianh): Really this should say "Error".
+  String toString() {
+    final List<String> parts = <String>[
+      if (exitCode != null) 'Exit code: $exitCode',
```

Comment:
Is this really helpful to the user?

Reply:
My concern is more that it's a little messier to have: `Exit code: 1. Error: ...`

Reply:
Yeah, you ask a fair question. We already print the exit code later in the typical tool exit workflow, so this could result in redundant noise for end-users. It would be nice if we could include it in the matcher's mismatch output; but, as far as I can tell, this would involve writing a bunch of custom matcher code, which I don't want to do. Maybe I will the next time I have troubles with `throwsToolExit` :)

I've reverted this part of the change (I did keep the `"Exception"` to `"Error"` change)